### ISSUE TITLE
Types of properties in 'extensions' in the property reference

### ISSUE THREAD

**Issue by javagl** [KhronosGroup/glTF#2158]
The [`mesh.primitive.attributes` schema](https://github.com/KhronosGroup/glTF/blob/35aec799d9a47f21dd3e651467c82d806b935693/specification/2.0/schema/mesh.primitive.schema.json#L13) declares
```
            "additionalProperties": {
                "$ref": "glTFid.schema.json"
            }
```
and the [`mesh.primitive.attributes` property reference](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#_mesh_primitive_attributes) shows this as 

> **Type of each property:** `integer`

In contrast to that, the [`extension`](https://github.com/KhronosGroup/glTF/blob/35aec799d9a47f21dd3e651467c82d806b935693/specification/2.0/schema/extension.schema.json#L9) schema declares
```
    "additionalProperties": {
        "type": "object"
    }
```
and the property reference for this - which is inlined into all property references, for example, the [`mesh.primitive.extensions` property reference](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#_mesh_primitive_extensions) - shows this as

> **Type of each property:** Extension

Shouldn't this "type of each property" be `object` instead?

(This came up via https://github.com/CesiumGS/wetzel/pull/68#discussion_r824755241 )

**Comment by lexaknyazev** [comment]
> Shouldn't this "type of each property" be `object` instead?

Good catch. Yes, members of the `extensions` object should be of `object` type (as `Extension` is not a JSON-Schema type at all).

**Comment by javagl** [comment]
One could argue about the term "JSON-Schema type" here, and rather say that in the _first_ case, the text should say

> **Type of each property**: glTFid

but with the caveat that `glTFid` does not appear in the property reference, but only in the schema reference. (It is listed as one of the "ignored types" when creating the property reference). 

Or to put it that way: **Iff** there was a structure with
```
"additionalProperties": {
    "$ref": "buffer.schema.json"
}
```
then it should not say `object`, but a link to [`buffer`](https://example.com) at this point. At least, I think that the current output is indeed the result of an attempt to exactly _do_ this sort of cross-linking between types. (IIRC, there is no case where this is relevant for glTF - here, it will always be `object` in these cases - but I'm trying to get a better understanding for wetzel right now...)
